docs(bg): update sample flight in preparing-mcdu.md (#939)
flybywiresim/docs: docs/pilots-corner/beginner-guide/preparing-mcdu.md

    Below is the current flight plan we are utilizing:

    ```title="Current Sample Flight Plan"
    EDDM/08L GIVMI6Q GIVMI Y101 ERNAS T161 DEBHI EDDF/07L
    ```

    Let's imagine for this example that there was no airway between `BCN` and `OKTEM`. The flight plan would look like the following:

    ```title="Example NO AIRWAY Flight Plan"
    EDDM/08L GIVMI6Q GIVMI DCT ERNAS DCT DEBHI EDDF/07L
    ```

    !!! note ""

>         The airwayS `Y101` and `T161` was replaced with `DCT` indicating from waypoint `GIVMI` proceed direct to `ERNAS` and after that proceed direct to `DEBHI`.

Reviewer: ```suggestion
        The airways `Y101` and `T161` was replaced with `DCT` indicating from waypoint `GIVMI` proceed direct to `ERNAS` and after that proceed direct to `DEBHI`.
```

    When you encounter this type of routing, there are a couple of ways to input this leg without the use of the airways page on the MCDU.

    One of the easiest ways is utilizing the lateral revision page. To do this, simply find the starting waypoint on your F-PLN page, which in this case is `GIVMI`.

    - Use the relevant LSK to select it. 
    - You will then be on the following page:

        ![lat-rev-fpln.png](../assets/beginner-guide/mcdu/mcdu-latrev.png){loading=lazy width=50%}

    - Type in `ERNAS` so it's visible in the scratchpad.
    - Use `LSK3R` to enter `ERNAS` as the next waypoint on your flight plan.

^^Planning the Arrival^^

For the purposes of this guide, we will pre-plan our arrival into EDDF via the `DEBHI1C` STAR into 07L.

!!! info "STAR"
    Standard Terminal Arrival Route

    Similar to the SID, these are procedures that are defined and published that take a flight from the last point in a route *(in our case `DEBHI`)* to the first point in the approach or the initial approach fix (IAF).

    Also see: [SIDS and STARS](../airliner-flying-guide/navigation.md#sids-and-stars)

> Find `EDDF` in green in the flight plan OR select `EDFF` in white under `DEST` using the corresponding LSK.

Reviewer: ```suggestion
Find `EDDF` in green in the flight plan OR select `EDDF` in white under `DEST` using the corresponding LSK.
```

* Select `ARRIVAL` using LSK1R
    * We will be shown the approaches available, designated by `Type` `Rwy`.
    * For this guide, we will shoot for an ILS to keep it simple.
* Use the vertical slew keys to find `ILS07LY` and select it using the corresponding LSK.
  * A designator such as Y or Z may be present. This suffix is to distinguish in the navigation database a difference between approaches to the same runway. While similar in nature they may have different minimums, allowed equipment, or other differences.
* Again, use the vertical slew keys to find the STAR for this flight `DEBHI1C` and select it using the corresponding LSK.

> * For the approach VIA Select `DF454` using the appropriate LSK. On the following page we can choose transitions if available, but for this flight we don't.

Reviewer 2: ```suggestion
* For the approach VIA, select `DF454`, using the appropriate LSK. On the following page, we can choose transitions, if available, but for this flight, we don't.
```

    The flight plan might contain so-called discontinuities. These are breaks in the flight plan and often separate two flight plan sections, like the SID and first in-route waypoint or the STAR and the APPR. They are also often inserted when the flight plan is modified.

>     Sometimes discontinuities are also part of a procedure to indicate that manual input is required (mostly clearance by ATC). The preceding legs are called MANUAL legs.In this flight plan we have a discontinuity between the STAR and approach procedure as shown below.

Reviewer: ```suggestion
    Sometimes discontinuities are also part of a procedure to indicate that manual input is required (mostly clearance by 
    ATC). The preceding legs are called MANUAL legs. These cannot and should not be deleted. See below link on how to 
    handle these. 
    
    In this flight plan we have a discontinuity between the STAR and approach procedure as shown below.
```

Author: I'm on GitHub but I might have forgotten to include it (and also forgot the title) -- but I meant to show a photo here of a MANUAL and F-PLN Disco just before we offload the user to a new page.

>     Sometimes discontinuities are also part of a procedure to indicate that manual input is required (mostly clearance by ATC). The preceding legs are called MANUAL legs.In this flight plan we have a discontinuity between the STAR and approach procedure as shown below.

Reviewer 2: ```suggestion
    Sometimes discontinuities are also part of a procedure to indicate that manual input is required (mostly clearance by ATC). The preceding legs are called MANUAL legs. In this flight plan, we have a discontinuity between the STAR and approach procedure, as shown below.
```

or add this correction accordingly to mavs suggestion.

    The flight plan might contain so-called discontinuities. These are breaks in the flight plan and often separate two flight plan sections, like the SID and first in-route waypoint or the STAR and the APPR. They are also often inserted when the flight plan is modified.

>     Sometimes discontinuities are also part of a procedure to indicate that manual input is required (mostly clearance by ATC). The preceding legs are called MANUAL legs. In this flight plan we have a discontinuity between the STAR and approach procedure as shown below.

Reviewer: ```suggestion
    Sometimes discontinuities are also part of a procedure to indicate that manual input is required (mostly clearance by ATC). The preceding legs are called MANUAL legs.  These cannot and should not be deleted. See below link on how to 
    handle these. 
    
    In this flight plan we have a discontinuity between the STAR and approach procedure as shown below.
```

    The flight plan might contain so-called discontinuities. These are breaks in the flight plan and often separate two flight plan sections, like the SID and first in-route waypoint or the STAR and the APPR. They are also often inserted when the flight plan is modified.

    Sometimes discontinuities are also part of a procedure to indicate that manual input is required
    (mostly clearance by ATC). The preceding legs are called MANUAL legs. 
    

>     **Discontinuities after a MANUAL leg cannot and should not be deleted. See below link on how to 

Reviewer 2: ```suggestion
    **Discontinuities after a MANUAL leg can not and should not be deleted. See below link on how to 
```
I would separate the cannot to emphasize the not.

^^VOR^^

The A32NX supports VOR autotuning when in range of a VOR before departure. You can verify this by checking the RADNAV page and seeing if the VOR frequency is already populated. You should verify the relevance of this VOR to your departure procedure and flight plan before takeoff.

> On this departure SID, we have routing instructions that rely on the Munich VOR `DMN` with a frequency of `116.0`

Reviewer 2: ```suggestion
On this departure SID, we have routing instructions that rely on the Munich VOR `DMN` with a frequency of `116.0`.
```

^^Departure ILS^^

When selecting the SID earlier in the flight plan section, the A32NX should have autopopulated the ILS/LOC frequency. If it hasn't, we can manually insert it for centerline guidance on take off.

> Our departure runway is EDDM/08L (runway 08 left) which has a frequency of `109.50`. When inputting a frequency, and we are in range of the ILS, it will auto populate the identifier and course. You could also enter the identifier `IMNE` and it would automatically fill in the frequency. You cannot enter both at the same time as this would result in a `FORMAT ERROR`. 

Reviewer 2: ```suggestion
Our departure runway is EDDM/08L (runway 08 left), which has a frequency of `109.50`. When inputting a frequency, and we are in range of the ILS, it will autopopulate the identifier and course. You could also enter the identifier `IMNE` and it would automatically fill in the frequency. You cannot enter both at the same time, as this would result in a `FORMAT ERROR`. 
```

* Using the keypad, type in `109.50` and press LSK3L to input it.

^^Arrival ILS^^

With an ILS or LOC approach selected, the arrival ILS frequency should be automatically tuned correctly whenever the aircraft is at climb phase or greater and within 250 NM of the destination. **Ensure** that we verify the ILS frequency when we reach the arrival phase of the flight - see [Approach and Landing (ILS)](landing.md).

> Remember, our arrival airport/rwy is `EDDF/07L` with ILS07L having a frequency of `110.30`. When inputting a frequency, and we are in range of the ILS, it will auto populate the identifier and course. There is no need to fill these fields. You could also enter the identifier `IFEL` and it would automatically fill in the frequency. You cannot enter both at the same time as this would result in a `FORMAT ERROR`. 

Reviewer 2: ```suggestion
Remember, our arrival airport/rwy is `EDDF/07L` with ILS07L having a frequency of `110.30`. When inputting a frequency, and we are in range of the ILS, it will autopopulate the identifier and course. There is no need to fill these fields. You could also enter the identifier `IFEL` and it would automatically fill in the frequency. You cannot enter both at the same time, as this would result in a `FORMAT ERROR`. 
```

^^VOR^^

The A32NX supports VOR autotuning when in range of a VOR before departure. You can verify this by checking the RADNAV page and seeing if the VOR frequency is already populated. You should verify the relevance of this VOR to your departure procedure and flight plan before takeoff.

On this departure SID, we have routing instructions that rely on the Munich VOR `DMN` with a frequency of `116.0`

> * Using the keypad, type in `116.0` and press LSK1L. This will autopopulate the identifier of the VOR when within range.
> * We can also set the desired course to track `080` and press LSK2L to input it. (This is for a departure on runway 08L)

Reviewer: ```suggestion
```

I would leave that out as we have autotuning now. Doing this manually is not something a beginner should do.

^^Departure ILS^^

When selecting the SID earlier in the flight plan section, the A32NX should have autopopulated the ILS/LOC frequency. If it hasn't, we can manually insert it for centerline guidance on take off.

> Our departure runway is EDDM/08l (runway 08 left) which has a frequency of `109.50`. When inputting a frequency, and we are in range of the ILS, it will autopopulate the identifier and course. There is no need to fill these fields.

Reviewer: ```suggestion
Our departure runway is EDDM/08L (runway 08 left) which has a frequency of `109.50`. When inputting a frequency, and we are in range of the ILS, it will auto populate the identifier and course. You could also enter the identifier `IMNE` and it would automatically fill in the frequency. You cannot enter both at the same time as this would result in a `FORMAT ERROR`. 
```

* Using the keypad, type in `109.50` and press LSK3L to input it.

^^Arrival ILS^^

With an ILS or LOC approach selected, the arrival ILS frequency should be automatically tuned correctly whenever the aircraft is at climb phase or greater and within 250 NM of the destination. **Ensure** that we verify the ILS frequency when we reach the arrival phase of the flight - see [Approach and Landing (ILS)](landing.md).

> Remember, our arrival airport/rwy is `EDDF/07l` with ILS07l having a frequency of `110.30`. When inputting a frequency, and we are in range of the ILS, it will autopopulate the identifier and course. There is no need to fill these fields.

Reviewer: ```suggestion
Remember, our arrival airport/rwy is `EDDF/07L` with ILS07L having a frequency of `110.30`. When inputting a frequency, and we are in range of the ILS, it will auto populate the identifier and course. There is no need to fill these fields. You could also enter the identifier `IFEL` and it would automatically fill in the frequency. You cannot enter both at the same time as this would result in a `FORMAT ERROR`. 
```

^^ECAM FOB^^

> Look at the upper ECAM and note the FOB indicated. Let's say that amount is `4631 kg`. When inputting the block fuel into the MCDU, it is referenced in "Tons" and we should round to the closest decimal point.

Reviewer 2: ```suggestion
Look at the upper ECAM and note the FOB indicated. Let's say that amount is `4631 kg`. When inputting the block fuel into the MCDU, it is referenced in "tons", and we should round to the closest decimal point.
```

^^SimBrief OFP^^

> We can use the planned block fuel stated on the OFP, which in this case is `4631 kg`.

Reviewer 2: ```suggestion
We can use the planned block fuel stated on the OFP, which in this case is `4631 kg`.
```

    ==}

> Our SID chart mentions that the TRANS ALT for this departure is 5000ft.

Reviewer 2: ```suggestion
Our SID chart mentions that the TRANS ALT for this departure is 5000 ft.
```
